docs/Course Wiki/ECE Course Offerings/ECE476.md

```diff
@@ -2,41 +2,58 @@
 tags:
 - Power
 ---
+
 # ECE476
 
+ECE476 (Power System Analysis and Control) is a 3-credit-hour senior/graduate level course that satisfies the Technical Electives requirement for ECE majors. This course is only offered in the Fall.
 
 
 ## Content Covered
 
 
+- Review of 3 phase systems from [ECE330](ECE330.md)
+- Transmission line parameters, modeling, and analysis
+- Transformer modeling
+- Load and generator modeling
+- Per unit representation of electrical systems
+- Network modeling
+- Control of power systems
+- Setting up and solving the power flow problem
+- Economic dispatch
+- Unbalanced system operation
+- Transient stability
 
-## Prerequisites
-
-
+ECE476 is starts with modeling basic components (transmission lines, generators, loads, and transformers) and builds up to modeling entire large scale electrical systems. ECE476 begins with basic three phase system analysis review (single phase equivalent, wye-delta conversion, complex power conservation etc.) from [ECE330](ECE330.md), but from a slightly different persepective. New content is then introduced when learning about transmission line modeling. While this content is somewhat familiar from [ECE329](ECE329.md), the approach is based on circuit analysis techniques rather than the rigorous physics of transmission line theory (no smith charts!). The class then delves into transformer modeling and the variety of different ways to create a three phase transformer. ECE476 then continues by going into load and generator modeling and touches on the differential equations that govern generator operation and control. Per unit representation is introduced during this unit to help simplify basic generator-load systems. Putting all of this together, students learn how to use admittance matrices to fully model a system with multiple loads, generators, transformers, and transmission lines. 
```

Comment:
Delete “is” in first sentence, capitalize “Smith”, maybe say “different three phase transformer connections”?